ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift

```diff
@@ -26,7 +26,7 @@ class BraveTranslateScriptHandler: NSObject, TabContentScript {
   static let scriptName = "BraveTranslateScript"
   static let scriptId = UUID().uuidString
   static let messageHandlerName = "TranslateMessage"
-  static let scriptSandbox = WKContentWorld.world(name: "BraveTranslateContentWorld")
+  static let scriptSandbox = WKContentWorld.defaultClient
```

Comment:
Chromium doesn't support custom content worlds, so this is swapped back to default